pkg/clustertree/cluster-manager/controllers/pod/root_pod_controller.go

```diff
@@ -825,6 +832,9 @@ func (r *RootPodReconciler) DeletePodInLeafCluster(ctx context.Context, lr *leaf
 	if err != nil {
 		if errors.IsNotFound(err) {
 			klog.V(4).Infof("Tried to delete pod %s/%s, but it did not exist in the cluster", leafPod.Namespace, leafPod.Name)
+			if cleanflag {
+				return cleanRootPodFunc()
```

Comment:
why make cleanRootPodFunc here instead of call function directly?

Reply:
Multiple calls to simplify statements